recipes/cubicinterpolation/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "0.1":
+    url: "https://github.com/MaxSac/cubic_interpolation/archive/v0.1.zip"
+    sha256: "470a245ebd63415d947d15405366f4e4ee3d25a804d56ed5fc0666423b1a7a08"
```

```suggestion
    url: "https://github.com/MaxSac/cubic_interpolation/archive/v0.1.tar.gz"
    sha256: "6f8d3a68222f5e2a4ef2048ff745cefe03df8acd38c0f90abd9159d25818a6d9"
```

Prefer tar.gz, it produces a smaller compressed file than .zip